### ISSUE TITLE
Change BOM and customer page colors

### ISSUE THREAD

**Issue by RachL** [openfoodfoundation/openfoodnetwork#10088]
## Description
**- As a:** enterprise or super admin users under 
**- On page:** BOM + customer page
**- I want to be able to do:** 

- [ ]  BOM AND customers : delete button is on the loose

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/471257c2-ef24-4b21-aee0-8bcf12ec4f50)

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/6cac4ff6-7325-401d-9244-a305a01c3ad7)

- [ ] BOM AND Customers : Change column selector to same design as the one in reports BUT use `#017a9a` for check boxes and "columns" text: 

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/39c02a32-7682-4d40-878a-782ce91d2b23)

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/55240cf2-bbc7-4000-ac74-a56d8c740af9)

- [ ]  BOM AND customers: use `#c1122B` for red

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/da763156-fcc5-40f1-9c4f-4b784583708f)

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/d59c9f95-adc9-4e22-ae0e-5cce5cf2378f)

- [ ] BOM : Fix alignment in FF

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/47ae5598-8f04-46b0-8c54-caa38a95164f)

- this grey is too light for square and placeholder, we need a darker one: 

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/b7df9c1c-4a79-40d9-8414-36d782c4ea4f)

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/e80c24bf-ae00-4404-8ed7-801434573176)

- [ ] green hover to fix (but probably will be when previous PR is merged

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/7cb8f585-531a-41aa-b5ec-435c75b6b30b)

- [ ]  Customers tags:  use `#017a9a` blue

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/9b55357a-9567-4597-a226-2b1c5243156b)

- | ] Customers : fix close the modal icon :D 

![image.png](https://images.zenhubusercontent.com/595e669a61b4ca3b0a737135/34aabed7-1020-45a6-8c97-46701ab00d50)

- [ ] customers: user darker grey? to check with new background color as well

![image](https://user-images.githubusercontent.com/6525576/205275164-c97b5229-8ee5-4d72-b4b2-0b3e8d1a1ff5.png)

## Acceptance Criteria & Tests

1. activate feature toggl: see colors are as described
2. Deactive feature toggl: spree colors are in da place

**Comment by RachL** [comment]
@jibees FYI I guess we are unblocked here as columns are a separate issue now?
